README.md

```diff
@@ -24,29 +24,35 @@ Yes, there is also a [Grunt Plugin](https://github.com/nwjs/grunt-nw-builder). F
 ## CLI Usage
 
 ```shell
-Usage: nwbuild [options] [path]
+Usage: 
+  nwbuild [options] [path] [-- <args>]
 
 Options:
-  -p, --platforms      Platforms to build, comma-sperated, can be: win32,win64,osx32,osx64,linux32,linux64   ['osx64', 'win32', 'win64']
-  -v, --version        The nw version, eg. 0.8.4                                             [default: "latest"]
-  -r, --run            Runs NW.js for the current platform                                   [default: false]
-  -o, --buildDir       The build folder                                                      [default: "./build"]
-  -f, --forceDownload  Force download of NW.js                                               [default: false]
+  -p, --platforms      Platforms to build, comma-sperated, can be: 
+                       win32, win64, osx32, osx64, linux32, linux64 or
+                       win, osx, linux                                       [default: <Current OS>]
+  -v, --version        The nw version, eg. 0.8.4                                 [default: "latest"]
+  -r, --run            Runs NW.js for the current platform                          [default: false]
+  -o, --buildDir       The build folder                                         [default: "./build"]
+  -f, --forceDownload  Force download of NW.js                                      [default: false]
   --cacheDir           The cache folder
-  --quiet              Disables logging                                                      [default: false]
-
+  --quiet              Disables logging                                             [default: false]
+  -- <args>            Pass custom arguments to the NW.js instance
+                       (-r, --run mode only)
 ```
+
 #### Run NW.js
-During development you can run NW.js with `nwbuild -r path/to/your/younwapp/`
+During development you can run NW.js with `nwbuild -r path/to/your/younwapp/ -- <args>`.
 
 ## Module Usage
 
 ```js
 var NwBuilder = require('nw-builder');
 var nw = new NwBuilder({
     files: './path/to/nwfiles/**/**', // use the glob format
-    platforms: ['osx64', 'win32', 'win64'],
-    version: '0.14.6'
+    platforms: ['osx64', 'win32', 'win64', 'linux32', 'linux64'],
+    version: 'latest',
+    argv: '[<nwjs_arg>, ... ]' // see nwjs docs for possible <nwjs_arg> values
```

Comment:
Still a string, should be `argv: ['<nwjs_arg>', ...]` (the quotes are around the `<nwjs_arg>` not the entire array)

Reply:
There should be no quotes anywhere actually. `'<x>'` is a literally <x>.